scheduler/playbooks/k8s_add_node.yml

```diff
@@ -141,3 +197,56 @@
     https_proxy: "{{ hostvars['localhost']['https_proxy'] }}"
     no_proxy: "localhost,127.0.0.1,{{ hostvars['localhost']['admin_nic_ip'] }},{{ hostvars['localhost']['omnia_config']['k8s_service_addresses'] }},{{ hostvars['localhost']['omnia_config']['k8s_pod_network_cidr'] }},{{ hostvars['localhost']['oim_hostname'] }},{{ hostvars['localhost']['domain_name'] }}"   # noqa: yaml[line-length]
     additional_no_proxy: "{{ hostvars['localhost']['user_no_proxy'] if hostvars['localhost']['no_proxy_input_status'] else '' }}"   # noqa: yaml[line-length]
+
+- name: Initiate add node in online mode
+  when:
+    - hostvars['127.0.0.1']['k8s_support']
+    - hostvars[groups['kube_control_plane'][0]]['k8s_installation_status']
+    - hostvars[groups['kube_control_plane'][0]]['k8s_new_nodes'] != ''
+    - hostvars['localhost']['k8s_offline_install'] is false
```

Comment:
we can use not hostvars['localhost']['k8s_offline_install']

Reply:
corrected

Reply:
corrected